image/templates/helm/stackrox-secured-cluster/templates/sensor.yaml.htpl

```diff
@@ -311,8 +311,7 @@ metadata:
     {{- $annotations := dict -}}
     {{- $_ := include "srox.getAnnotationTemplate" (list . "helm-hook_secret" $annotations) -}}
     {{- include "srox.annotations" (list . "secret" "helm-effective-cluster-name" $annotations) | nindent 4 }}
-stringData:
-  cluster-name: |
-    {{- ._rox.clusterName | nindent 4 }}
+data:
+  cluster-name: {{ ._rox.clusterName | b64enc }}
```

Comment:
How about the following, which should keep the output human-readable?

```suggestion
stringData:
  cluster-name: |
    {{- quote ._rox.clusterName | nindent 4 }}
```

Reply:
I tried that, but it does not work. When installed with legacy bundle, the name would include the quotes and sensor would not be able to connect to central because of the name mismatch.